operations/bills.md

```diff
@@ -88,6 +88,7 @@ Returns all bills, optionally filtered by customers, identifiers and other filte
             "CustomerId": "fe795f96-0b64-445b-89ed-c032563f2bac",
```

Comment:
`CustomerId` and `CompanyId` are now deprecated, so they should be removed from the sample JSON, replaced by `AssociatedAccountsIds`.